src/utils/funksjonerForAmplitudeLogging.ts

```diff
@@ -51,13 +113,18 @@ export const loggBedriftValgtOgTilganger = (
     if (org.altinntilgang.varigLønnstilskudd) {
         tilgangskombinasjon += 'varig lønnstilskudd';
     }
-
-    const tilgangsinfo: EventProps = {
+    const eregInfo = await hentInfoFraEreg(org);
```

Comment:
Jeg sitter og lurer på om vi burde passe på at funksjonen returnerer med en gang, og at loggingen skjer på siden (f.eks. ved hjelp av `setTimeout(..., 0)`. ). Er redd for at treghet mot ereg fører til treghet i bytte av virksomhet. Mulig det ikke skjer, da?

Reply:
Er jo asynch, så den blokkerer vel ikke noe annet som skjer på siden?